Makefile

```diff
@@ -438,25 +438,25 @@ build/kuma-tcp-echo/linux-amd64:
 
 docker/build: docker/build/kuma-cp docker/build/kuma-dp docker/build/kumactl docker/build/kuma-injector docker/build/kuma-init docker/build/kuma-prometheus-sd docker/build/kuma-tcp-echo
 
-docker/build/kuma-cp: build/artifacts-linux-amd64/kuma-cp/kuma-cp
+docker/build/kuma-cp: build/kuma-cp/linux-amd64
```

Comment:
Why these renames required? I believe  `build/artifacts-linux-amd64/kuma-cp/kuma-cp` should be consistent with filesystem

Reply:
Thanks, you are right!
I reverted the change and added comments so I don't make the same mistake again.